media/css/products/vpn/download.scss

```diff
@@ -6,28 +6,242 @@ $font-path: '/media/protocol/fonts';
 $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
-@import 'includes/lib';
 
-.vpn-download-header {
-    text-align: center;
+.vpn-downloads {
+    background-color: $color-marketing-gray-10;
 }
 
-.vpn-download-list {
-    text-align: center;
-    margin-top: $layout-lg;
+.vpn-downloads,
+.vpn-privacy {
+    .mzp-c-split {
+        .mzp-c-split-container {
+            min-width: unset;
+        }
+
+        @media #{$mq-md} {
+            padding: $spacing-lg 0;
+            @include text-body-xl;
+        }
+    }
+}
+
+.vpn-download-primary-platform,
+.vpn-download-secondary-list {
+    .platform-body {
+        h2 {
+            @include font-base;
+            @include text-body-xl;
+        }
+
+        p {
+            color: $color-marketing-gray-60;
+        }
+    }
+}
+
+.vpn-download-options {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-row-gap: $spacing-2xl;
```

Comment:
Issue (non-blocking): this could maybe use some column gap too? Things can get a little squished between the primary and secondary blocks:

![image](https://user-images.githubusercontent.com/400117/231713962-02b1a201-6aa7-4f48-b05b-07d4ef0e1e82.png)